src/js/components/search/SearchOption.jsx

```diff
@@ -47,62 +47,59 @@ export default class SearchOption extends React.Component {
     }
 
     render() {
+        let disabledStatus = false;
+        let comingSoon = null;
+        let comingSoonModule = (
```

Comment:
This needs to be `const` to pass the linting (getting a green checkmark in Github instead of the red X)